### DISCUSSION TITLE
SSR & AisRefinementList?

### DISCUSSION THREAD

**Discussion by Krillko** [nuxt-modules/algolia#173] [Q&A]
I'm trying to add AisRefinementList to a SSR rendered page. 
If I just add 
```
return h(AisInstantSearchSsr, null, () => [
                        // Include any vue-instantsearch components that you use including each refinement attribute
                        h(AisHits),
                        h(AisRefinementList),
                    ]);
```
it breaks ssr of the hits. 

Could you be more specific what "including each refinement attribute" means and the syntax for it? I'm trying to add "categoryFilters" `<ais-refinement-list attribute="categoryFilters" />`

**Comment by vilhelmjosander** [comment]
I'm facing the same issue. The documentation is not really clear on how to do this unfortunately.

**Comment by Baroshem** [comment]
MAybe this will help you https://github.com/nuxt-modules/algolia/issues/164#issuecomment-1738626323

**Comment by Baroshem** [comment]
Hey guys, one of the contributors created this stackblitz with a demo of using SSR in AisRefinementList. maybe it will help you :)

https://stackblitz.com/github/plexus77/nuxt-3-algolia-ssr?file=app.vue

**Comment by Rigo-m** [comment]
Hi all, current SSR Implementation is so bad for many reasons, and very verbose.
Here is my take on how to do it (almost) properly.
https://github.com/Rigo-m/nuxt-ssr-algolia-example
I hope that in the future algolia will step up its vue game.

**Reply by Baroshem** [reply]
Hey @Rigo-m 

I have answered you in #187 

Regarding Algolia stepping up in Vue game, to be honest I dont think it will happen. Majority of their customers is React based. Vue is more like community play.
